crates/rolldown/tests/rolldown/errors/parse_error/invalid_define_key/artifacts.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Errors
+
+## PARSE_ERROR
+
+```text
+[PARSE_ERROR] Error: Parse failed, got: "`6` is not an identifier."
```

Comment:
https://app.graphite.dev/github/pr/rolldown/rolldown/2884/feat-use-better-diagnostic-for-ReplaceGlobalDefinesConfig#comment-PRRC_kwDOKVE1Gs5ul2Q6